lib/definition.js

```diff
@@ -21,18 +21,17 @@
   (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
   THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-"use strict";
 
-const Variable = require("./variable");
+import Variable from "./variable.js";
 
 /**
- * @class Definition
+ * @constructor Definition
```

Comment:
Yeah, it's the current enforcement by `eslint-plugin-jsdoc`. The preference can be overridden in your eslint config.

Reply:
Sorry, I was mistaken about it being the new default. It appears your config must be changing it. And frankly, I agree with the change. (We currently prefer the tag name not declared as an alias by jsdoc.)

FWIW, what follows are my reasons for favoring your current config's approach of favoring `@constructor`...

Those `@class`/`@constructor` tags should only be necessary when jsdoc can't detect the surrounding type and name. In all the cases in `eslint-scope`, it appears straightforward, so I think the whole tag is redundant. (Even [`@classdesc`](https://jsdoc.app/tags-classdesc.html), which allows an accompanying description, would be redundant here since one could just describe the class as free form text of the block comment when the fact of it being a class and its name were both auto-discoverable from context.)

So, the `function aConstructorFunction () { this.a = 5; }` type of "class" would be the only one needing the tag, but I think that `@class` seemed to muddy the waters with ES6 classes whereas `@constructor` fits.

`@class` therefore might only be useful iI think n a situation where one wanted to document a variable as being a class outside of the context where it was defined, and for those wishing to do so, they'd have to be set in `settings.jsdoc.tagNamePreference`. See the [jsdoc/check-tag-names](https://github.com/gajus/eslint-plugin-jsdoc/#eslint-plugin-jsdoc-rules-check-tag-names).

Reply:
Ah ok, let's stick with what the config says. No need to fight another decision.